plugin/src/main/java/net/dzikoysk/funnyguilds/guild/RegionManager.java

```diff
@@ -96,6 +104,34 @@ public boolean isInRegion(Location location) {
         return this.findRegionAtLocation(location).isPresent();
     }
 
+    public boolean isAnyPlayerInRegion(Region region, Collection<UUID> ignoredUuids) {
+        if (!pluginConfiguration.regionsEnabled) {
+            return false;
+        }
+
+        FunnyBox box = region.toBox();
+
+        for (Player player : Bukkit.getOnlinePlayers()) {
+            if (!box.contains(player.getLocation().toVector())) {
+                continue;
+            }
+
+            if (!ignoredUuids.contains(player.getUniqueId())) {
```

Comment:
Ja bym to sprawdzał najpierw, zanim się posprawdza po kolei wszystkie te pozycje